styles.css

```diff
@@ -137,16 +137,27 @@ header img {
 	min-width: 300px;
 }
 
+.about a:link, .about a:visited {
+	color: #e02c5e;
+	font-weight: bold;
+	
+}
+
 .contact {
 	background-color: #6a6c6e;
 	margin: 0;
 	padding: 20px 60px;
 }
 
 .social-link-icons {
-	width: 200px;
 	display: flex;
-	justify-content: space-between;
+	justify-content: flex-start;
+}
+
+.social-link-icons-container {
+	display: flex;
+	align-items: center;
+	margin: 0 20px 10px 0;
```

Comment:
its best practice to use the gap option from flex instead of padding. At the bottom you're also layering padding which is generally frowned on, put all your bottom padding only on the section and not on the last div